pkg/plugin/conntrack/conntrack_linux.go

```diff
@@ -0,0 +1,129 @@
+// Copyright (c) Microsoft Corporation.
```

Comment:
Can you add a `Snapshot` function that prints the current state of `conntrack` map? I want to validate it's updated as expected and can be used to debug retina issues in future.

Reply:
i have some debug logs that would print out entries in conntrack every GC cycle, are you saying that we should have a CLI that should be able to do that instead?

Reply:
Yeah, add that function so that we can add it to a CLI debug command. The snapshot of `conntrack` can be useful to debug bpf issue.

Reply:
![image](https://github.com/user-attachments/assets/2a2d5ef7-18e7-4760-9108-03b1a607a1f9)would something like this useful? planning to pivot from this to adding a debug CLI as well